pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_identifying-module-types.adoc

```diff
@@ -1,47 +1,54 @@
 [id='identifying-module-types_{context}']
-= Identifying {ProductShortName} module types
-:pantheon-module-type: REFERENCE
+= Adding the module type to modules in {ProductShortName}
+:_module-type: REFERENCE
 
 [role="_abstract"]
-In {ProductShortName}, each module must be identified by type: concept, procedure, or reference.
+In {ProductShortName}, you can search and filter modules  by type: concept, procedure, or reference. You declare a module's type by adding an identifier to each module.
 
-For information about module types, see https://redhat-documentation.github.io/modular-docs/#writing-mod-docs[Types of modules in Red Hat documentation].
+Use one or both of the following identifiers to declare the module's type:
 
-Identify the module type by using either one or both of the following:
-
-* The correct attribute after the module's title
-* The correct prefix in the module's name
-
-Using the correct attribute (preferred method)::
+* The `:_module-type:` attribute after the module's title
+* The corresponding prefix in the module's file name
 
+Attribute (preferred method)::
 Enter the correct attribute in the line immediately following the module's title, choosing among the following:
-
-* `:pantheon-module-type: CONCEPT`
-* `:pantheon-module-type: PROCEDURE`
-* `:pantheon-module-type: REFERENCE`
-
++
+* `:_module-type: CONCEPT`
+* `:_module-type: PROCEDURE`
+* `:_module-type: REFERENCE`
++
 ====
 .Example
 
-`= Identifying {ProductShortName} module types :pantheon-module-type: REFERENCE`
+`= Identifying {ProductShortName} module types
+:_module-type: REFERENCE`
 
 ====
 
-Using the correct prefix::
+Prefix::
+Begin the file names of modules with one of the following prefixes, followed by either an underscore or a hyphen:
++
+* `con`
+* `proc`
+* `ref`
++
+====
+.Examples: A reference module
 
-Begin the file names of modules with the correct prefix, choosing among the following:
+Underscore after the prefix:
 
-* `con_`
-* `proc_`
-* `ref_`
+`**ref_**identifying-module-types.adoc`
 
-====
-.Example
+Hyphen after the prefix:
 
-`ref_identifying-module-types.adoc`
+`**ref-**identifying-module-types.adoc`
 
 ====
 
 How {ProductShortName} {ProductNumber}  checks identifiers::
```

Comment:
Do you think this will be better as a second level heading (discrete) rather than a part of the def list? 

- Attribute (preferred method)
- Prefix
- How Pantheon 2 checks identifiers

The first two are parallel, but not the third.

(I know this was already there, but thought I'd comment anyway :) )

Reply:
Yes, you're right. I actually moved this section to precede the detail explanation of the attribute and the prefix.